fastlane/lib/fastlane/actions/ssh.rb

```diff
@@ -6,7 +6,7 @@ module SharedValues
     end
 
     class SshAction < Action
-      def self.ssh_exec!(ssh, command)
+      def self.ssh_exec!(ssh, command, log)
```

Comment:
Should `log` have a default value of `nil` here so that it won't break other/older calls to this method that do not pass in a `log` parameter?